### ISSUE TITLE
Too many accounts to fit on the screen

### ISSUE THREAD

**Issue by 20Fourteen** [winauth/winauth#198]
**What version of WinAuth?**
Latest

**What system/version (e.g. Windows 7)?**
8

**How many authenticators do you have? If one, what type (e.g. Google
Authenticator/Battle.net)?**
More than WinAuth can fit on the screen

**Do you use a password?**
Yes

**Do you use the "Encrypt to only be useable on this computer" protection?**
N/A

**What steps will reproduce the problem?**
1.Add many accounts

**If you are able to load WinAuth, please click the cog icon and choose
"About...". Click the Diagnostics button and copy the contents of the field
(beginning with "--BEGIN PGP MESSAGE---") into a file and attach to this
issue.**
N/A

**Please provide any additional information:**
Please enable horizontal stacking after 10 accounts, because I have many accounts (15+) and they will not fit on the screen. Thanks

Check out the screenshot: 
![image](https://cloud.githubusercontent.com/assets/8763290/7105775/9eb404a6-e121-11e4-8a10-d5e7f45c854d.png)

**Comment by winauth** [comment]
While maybe adding grouping or tree type structure in the future, right now you should turn off Auto Size and resize it manually. I have added events for mouse wheel to make scrolling easier.

**Comment by 20Fourteen** [comment]
If you add scrolling then also ensure that the application has a maximum height, so that the scrolling can be used effectively.

At the moment the latest beta version does not fix the problem. The app is higher than the desktop and the lower part appears beneath the taskbar.

**Comment by winauth** [comment]
It is already supposed to only be a maximum of 62% of the screen height if AutoSize is on. Are you able to get to the options button so you can turn AutoSize off and on again. Does that let you resize it? Or is your config file unencrypted, so you can see what you have set for the Height option?

**Comment by 20Fourteen** [comment]
Thanks for your comment, I went and disabled auto-size but nothing changed. Then I proceeded to decrypt (remove protection) the XML file and changed the height value to 420, then I reopened the app and could scroll up and down.

Now when I turn on auto-size it will expand to 62% of the screen, so there might be a bug when adding multiple accounts continuously that the size expands all the way and does not invoke the maximum height and scrolling.

So we have a workaround, and all is good - thanks for the program.
